src/cityreader/cityreader.py

```diff
@@ -1,10 +1,21 @@
+import csv
 # Create a class to hold a city location. Call the class "City". It should have
 # fields for name, lat and lon (representing latitude and longitude).
+class City:
+    def __init__(self, name, lat, lon):
+        self.name = name
+        self.lat = lat
+        self.lon = lon
+
+    def __str__(self):
+        return f"{self.name}, {self.lat}, {self.lon}"
```

Comment:
Vegas = City("Las Vegas", 100, -100)

print(Vegas) = > < 0x09871vbs at 12893712 >

Override this behavior with string dunder